README.md

```diff
@@ -148,11 +148,19 @@ usage does not exceed the threshold.
 
 ## DoctorKafka UI 
 
-DoctorKafka uses an embedding Jetty server to provide a web UI. The following is the screenshot from a demo:
+DoctorKafka uses Dropwizard assets to provide a web UI. The following is the screenshot from a demo:
```

Comment:
The term `assets` can be confusing to some readers.  How about  use the following to provide a url link to the readers? 

    [dropwizard-core module](https://www.dropwizard.io/0.9.2/docs/manual/core.html)

Reply:
Updated with [dropwizard-core module](https://www.dropwizard.io/1.3.5/docs/manual/core.html#serving-assets)

Reply:
do we need to add `#serving-assets` in the url?   DoctorKafka uses jetty sever, and also uses HttpServlet to render html.  pointing to https://www.dropwizard.io/1.3.5/docs/manual/core.html#serving-assets can cause confusion.

Reply:
Dropwizard assets is not a part of the core package, therefore I think it would be good to specify it explicitly. Without the asset package the UI can't be rendered. There are other methods that can be used to render HTML with dropwizard like server-side rendering. Asset rendering affects the application context path which would otherwise point to '/' (root)

Reply:
how about the following?  clicking on "dropwizard-core module" and seeing "serving asserts" will be confusing. 

    [dropwizard-core module](https://www.dropwizard.io/0.9.2/docs/manual/core.html) and [serving assets](https://www.dropwizard.io/1.3.5/docs/manual/core.html#serving-assets)

Reply:
That works, updated PR